CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - [#1002](https://github.com/Azure/azure-storage-fuse/issues/1002) In case version check fails to connect to public container, dump a log to check network and proxy settings.
 - [#1006](https://github.com/Azure/azure-storage-fuse/issues/1006) Remove user and group config from logrotate file.
 - [csi-driver #809](https://github.com/kubernetes-sigs/blob-csi-driver/issues/809) Fail to mount when uid/gid are provided on command line.
+- [#1032](https://github.com/Azure/azure-storage-fuse/issues/1032) CLI params updation fix in `mount all` command.
```

Comment:
could you please explicitly mention here what the issue was - so for example "correctly parse and gracefully terminate on incorrect input parameters"